include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -70,34 +70,24 @@ namespace dht
 		void reset();
 	};
 
-	struct TORRENT_EXPORT dht_immutable_item
+	struct TORRENT_EXPORT dht_immutable_data
```

Comment:
they're still public and still widen the interface to the library. They're just duplicated, I don't think that really adds that much. One potential benefit I can imagine is that you may not want to actually store the bloom filter, but rather some kind of popularity metrics. Or perhaps nothing at all, and just let the popularity be re-recorded